### PR TITLE
Feature/extract dau

### PR DESCRIPTION
**Pull request by KarenAJ**

### CODE REVIEW

**Review by jdiouf — APPROVED** [review]

**Review by micson-us — APPROVED** [review]
PR reviewed

**Comment by mattdrayer** [comment]
@KarenAJ please be sure to prune (delete) working branches after merging.

File: app/core/management/commands/map_dau.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        upload_bucket_name = os.environ['UPLOAD_BUCKET_NAME']
+
+        # dau_path = 's3://dauxsr1/CSV_DAU_Consolidated.csv'
+        dau_path = 's3://%s/%s/' % (bucket_name,data_file_name)
+
+        # upload_path = '/opt/app/openlxp-xia-dau/core/management/commands/CSV_DAU_Result.csv'
+
+        dirpath = os.getcwd()
+        output_path = os.path.join(dirpath, 'output.csv')
+        # upload_path = '/opt/app/temp/csv_dau_result.csv'
```
Comment (mattdrayer) [inline app/core/management/commands/map_dau.py:25]: For merges to the main branch we should avoid including commented-out code unless there is a need for it.  Also these environment-specific values should be moved to a configuration file.

File: app/core/management/commands/map_dau.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        # json connect
+
+        # dau_json = s3.Object('dauxsr1', 'schema.json')
+        dau_json = s3.Object(bucket_name, schema_file)
+        dau_json_content = dau_json.get()['Body'].read().decode('utf-8')
+        data_dict = json.loads(dau_json_content)
+        # self.stdout.write(data_dict)
```
Comment (mattdrayer) [inline app/core/management/commands/map_dau.py:32]: debug statements should either be removed, or better, converted into [logging statements](https://docs.djangoproject.com/en/3.1/topics/logging/) that can provide insight into how the workflow executes and processes information at runtime.

File: app/core/management/commands/map_dau.py (hunk 1 above)
Comment (KarenAJ) [reply app/core/management/commands/map_dau.py:25]: Sure, got it. Will make the changes

**Pull request merged**